certified-connectors/Ministry Brands ParishSOFT/README.md

```diff
@@ -0,0 +1,54 @@
+# ParishSOFT Family Suite API
+Ministry Brands ParishSOFT Family Suite is an integrated church management software solution that gives you access to everything you need for your ministry to run smoothly. Pastors, staff, and parish administrators can streamline the behind-the-scenes administrative work. Rather than having a different software or online platform for membership, online giving, communications, and volunteer management, Ministry Brands ParishSOFT Family Suite cohesively ties all the tasks into one system. This connector uses the Family Suite APIs to make it easy to interact with information captured using the Family Suite of applications. You can utilize various actions in the connector to easily extract data from Family Suite and integrate with other applications.
+
+# Publisher
+Ministry Brands ParishSOFT
+
+# Prerequisites
+In order to use this connector, you will need the following:
+
+- An active Family Suite License
+- Family Suite API Key
+- Microsoft Power Apps or Power Automate plan
+
+# Supported Operations
+
+## Actions
+### Constituents
+- Search Constituents
+- Get a single Constituent record
+
+### Families
+- Search families
+- Get a single family record
+- Get a list of family members
+- Get a list of family ministries
+- Get a list of family group lookup values
+- Get a list of family work groups
+- Update family contact information
+- Update family auto fill values
+- Get a list of family changes
+
+### Members
+- Search members
+- Get a single member record
+- Get a list of member sacraments
+- Get a list of member work groups
+- Get a list of member status lookup values
+- Get a list of member contact information
+- Update member contact information
+
+### Offering
+- Get a list of organization funds
+- Get a list of organization givers
+- Get a list of family contribution summaries
+
+### Organizations
+- Search organizations
+- Get a single organization record
+
+## Obtaining Credentials
+Please contact ParishSOFT to obtain the necessary credentials for access.
+
```

Comment:
Please add deployment instructions. 
Refer the link below https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Added Deployment Instructions to README.md file. Updated pull request with additional commit.